### ISSUE TITLE
IgxToast `show` should take a message param

### ISSUE THREAD

**Issue by grosch** [IgniteUI/igniteui-angular#7156]
It's pretty common to use one IgxToast but want to support multiple messages.  Instead of having to do this:

```typescript
this.toast.message = "xyz"
this.toast.show()
```

it would be nicer to just do

```typescript
this.toast.show("xyz")
```

**Comment by radomirchev** [comment]
@grosch , the feature request is in the list for the current milestone. You could follow the milestone progress on the [Roadmap board](https://github.com/orgs/IgniteUI/projects/122) or in the [Roadmap file](https://github.com/IgniteUI/igniteui-angular/blob/master/ROADMAP.md)

**Comment by zdrawku** [comment]
Hello @grosch, with 9.1.0 we've added a functionality for displaying various content into the toast component through content projection. You can now place your message or any html element in the toast content which gives you more powerful and diverse way of toast content handling. Given that, I wanted to let you know that we are not planning to add a `message` param to the `show` method. Let me know if you have any questions or concerns.

We are going [to update ](https://github.com/IgniteUI/igniteui-docfx/issues/2032)the official topic as well in order to address that change.

**Comment by grosch** [comment]
That's doesn't really accomplish anything different though.  So now I'll put a variable in the toast part of the HTML, but I still have to then take two steps to display it.  I set the value, and then I call show.

If I only have one message the toaster would display what you're doing help.  However, when it's a reusable item, which can display multiple messages depending on state, I'm still at a two step process.

**Comment by zdrawku** [comment]
I understand your point, we always strive to have simple and clear API, trying to limit having a couple of ways to achieve equivalent functionalities. After further discussion with @radomirchev we've decided to add this capability.

Adding this FR to next sprint

**Comment by zdrawku** [comment]
Hey @grosch, the new functionality will be available with the next product release 10.0.3.
